corehq/apps/accounting/migrations/0066_remove_display_condition_on_nodeset_toggle.py

```diff
@@ -0,0 +1,33 @@
+from django.db import migrations
+
+from corehq.util.django_migrations import skip_on_fresh_install
+from corehq.toggles.models import Toggle
+from couchdbkit import ResourceNotFound
+
+TOGGLE_SLUG = "display_condition_on_nodeset"
```

Comment:
I wonder if there is a better place than `accounting` app to do this, since this cleanup doesn't belong to accounting app.

@snopoke Is there any recommended way to cleanup Couch docs other than through Django migrations?

Reply:
@Charl1996 

Sorry, I know I suggested this migration, but I think a better way to do this would be to delete them in lumps, say after every 10 or so. For now, we can keep the toggle in a TO_BE_DELETED_TOGGLES under corehq.toggles and create a delete migration after 10 or so toggles.

Reply:
@sravfeyn Yeah, that's not a bad idea. I created [this doc](https://docs.google.com/document/d/1c_VS3kKjq3BC6WU_k0sKTulJxKePRjMC5oyRPFD8Qpg/edit#) so we have a place to keep track of the toggles that's been deprecated and needs removing. If there's already something in place, feel free to ignore.

Reply:
It's also fine to just leave the couch docs in the DB. There are very few of them and they are small.